### ISSUE TITLE
Local volume Upload doesn't start resize

### ISSUE THREAD

**Issue by fly1ted** [engram-design/ImageResizer#52]
This validates an asset on [EVENT_BEFORE_HANDLE_FILE](https://github.com/craftcms/cms/blob/master/src/elements/Asset.php#L1240) while `kind` is set [later](https://github.com/craftcms/cms/blob/master/src/elements/Asset.php#L1247). Thus `$asset->validate()` returns an err: `Kind cannot be blank.`
https://github.com/verbb/image-resizer/blob/8289d1dd9a960e425724f8d6baf187b7fa0234f2/src/services/Service.php#L37

I try to upload an asset via `/admin/assets` not from an Entry.

```
PHP version	7.3.4
OS version	Linux 4.4.0-134-generic
Database driver & version	PostgreSQL 10.7
Image driver & version	Imagick 3.4.3 (ImageMagick 6.9.7-4)
Craft edition & version	Craft Solo 3.1.29
Yii version	2.0.19
Twig version	2.11.2
Guzzle version	6.3.3
Imagine version	0.7-dev
Plugins
Asset Rev	6.0.2
Control Panel CSS	2.1.0
Dark Mode	1.1.0
Delete Entry Versions	2.0.0
Image Resizer	2.0.5
Maps	3.4.11
SEOMate	1.0.2
Super Table	2.1.20
```

**Comment by engram-design** [comment]
I'm not seeing this in Craft 3.3.x? The only way I can reproduce this is via a front-end form, not from the control panel (a field on an entry, or the main assets area)

**Comment by engram-design** [comment]
Should be fixed in [2.0.6](https://github.com/verbb/image-resizer/releases/tag/2.0.6).
